code/style.css

```diff
@@ -1,4 +1,207 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/*These style the main elements of entire website*/
+
+body {
+  margin: 0;
+  font-family: 'Cormorant', serif;
+  font-family: 'Cormorant Garamond', serif;
+  font-family: 'Spectral', serif;
+  background-image: url("background.png");
 }
+
+main {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+h1 {
+  font-size: 60px;
+  margin-top: 25px;
+  margin-bottom: 0;
+}
+
+h3 {
+  font-style: italic;
+}
+
+h4 {
+  font-size: 15px;
+  margin: 0;
+
+}
+
+h5 {
+  font-size: 12px;
+  margin: 0;
+}
+
+/*These style the header and its content*/
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+nav {
+  width: 40%;
+}
```

Comment:
The comment /**/ you wrote describing what html section these CSS styles rule made it clear and also simplified readability, nice! However I believe writing ```nav ul {}``` could increase quick readability even more, also in case if there were to be other ul in the page.

Reply:
Agree! I really tried to implement this but did not manage without destroying the flex. Will continue to try!